config/terraform/aws/cloudfront.tf

```diff
@@ -10,7 +10,7 @@ resource "aws_cloudfront_distribution" "key_retrieval_distribution" {
     custom_origin_config {
       http_port              = 80
       https_port             = 443
-      origin_protocol_policy = "https-only"
+      origin_protocol_policy = "http-only"
```

Comment:
Missed this earlier, this should not be changed to http

Reply:
thanks, I talked to @maxneuvians I thought origin was client side, not destination, Ive since fixed it